notificationworkerlambda/src/main/scala/com/gu/notifications/worker/SenderRequestHandler.scala

```diff
@@ -78,6 +87,7 @@ trait SenderRequestHandler[C <: DeliveryClient] extends Logging {
         deliverIndividualNotificationStream(individualNotifications)
           .broadcastTo(
             reportSuccesses(chunkedTokens, sentTime, functionStartTime, sqsMessageBatchSize),
+            reportLatency(chunkedTokens, chunkedTokens.notificationAppReceivedTime.getOrElse(functionStartTime)), // FIXME: remove this fallback after initial deployment
```

Comment:
As mentioned before, this time is optional for now so that any messages which are already in the queue (at deployment time) can still be processed. 

We'll be able to make this mandatory in a follow-up PR, which will allow us to remove this hack.